### ISSUE TITLE
pip: No distributions found for dbus

### ISSUE THREAD

**Issue by ethantkoenig** [slgobinath/SafeEyes#223]
When I trying install `safeeyes` via `pip` on Ubuntu, I get the following error:

```
$ sudo pip3 install safeeyes
Downloading/unpacking safeeyes
  Downloading safeeyes-2.0.1.tar.gz (211kB): 211kB downloaded
  Running setup.py (path:/tmp/pip_build_root/safeeyes/setup.py) egg_info for package safeeyes
    
Downloading/unpacking babel (from safeeyes)
  Downloading Babel-2.5.1-py2.py3-none-any.whl (6.8MB): 6.8MB downloaded
Downloading/unpacking dbus (from safeeyes)
  Could not find any downloads that satisfy the requirement dbus (from safeeyes)
Cleaning up...
No distributions at all found for dbus (from safeeyes)
Storing debug log for failure in /root/.pip/pip.log
```

It seems that dbus (which is listed as a requirement in `setup.py`) is not a PyPI package. Should it instead be [dbus-python](https://pypi.python.org/pypi/dbus-python)?

**Comment by slgobinath** [comment]
Today I faced this problem and looking at the issue.  Obviously, the `dbus` is not a Python module but I couldn't find the proper module for Python 3.

I also had problems with `gi` installed using `pip`.

@FedericoCeratto please have a look at this. If these dependencies are not required for Debian, shall we remove it?

FYI: This dependency was added in https://github.com/slgobinath/SafeEyes/pull/216.

Until we resolve this issue, I hide the 2.0.1 from PyPi.

**Comment by slgobinath** [comment]
Fixed in 2.0.2.
